ci/functional/job_cleanup.sh

```diff
@@ -13,17 +13,9 @@ if $TEST_RPMS; then
         tar -C install/lib/daos/TESTING/ -xzf -
 fi
 
-rm -rf install/lib/daos/TESTING/ftest/avocado/job-results/*/*/html/
-
-# Remove the latest avocado symlink directory to avoid inclusion in the
-# jenkins build artifacts
-rm -f install/lib/daos/TESTING/ftest/avocado/job-results/latest
-
-arts="$arts$(ls ./*daos{,_agent}.log* 2>/dev/null)" && arts="$arts"$'\n'
-arts="$arts$(ls -d \
-   install/lib/daos/TESTING/ftest/avocado/job-results/* 2>/dev/null)" && \
-  arts="$arts"$'\n'
-if [ -n "$arts" ]; then
-    # shellcheck disable=SC2046,SC2086
-    mv $(echo $arts | tr '\n' ' ') "${STAGE_NAME}/"
-fi
+if [ -e "install/lib/daos/TESTING/ftest/avocado/job-results/${STAGE_NAME}" ]; then
```

Comment:
(lint) Couldn't find 'fi' for this 'if'. [SC1046]
(lint) Couldn't parse this if expression. Fix to allow more checks. [SC1073]